nav2_smac_planner/src/node_lattice.cpp

```diff
@@ -69,6 +98,41 @@ void LatticeMotionTable::initMotionModel(
 
   // TODO(Matt) populate num_angle_quantization, size_x, min_turning_radius, trig_values,
   // all of the member variables of LatticeMotionTable
+
+  num_angle_quantization = latticeMetadata.number_of_headings;
+  num_angle_quantization_float = static_cast<float>(num_angle_quantization);
+  min_turning_radius = latticeMetadata.min_turning_radius;
```

Comment:
You can just straight up remove `bin_size` from this object since it doesn't make sense in our context with non-regular sizes

Reply:
Done

Reply:
Done, had to comment out a few lines in `getAnalyticPath()` that used the bin size.